src/aboutcomponents/overview.js

```diff
@@ -12,22 +11,22 @@ const overview = () => {
 
         <div className="quoteDiv">
           <h3 className="quoteText">We said ‘we think we can do more, more than just solar.’ We knew we could make a greater impact with a broader understanding of sustainability. That’s when we made the decision to create Cornell University Sustainable Design.
-            <br></br><br></br> – Jeremy Blum '13 </h3>
+            <div className="founder">
+              – Jeremy Blum '13 </div> </h3>
```

Comment:
Nit: if you're going to have elements that span multiple lines/nested tags, please try to keep the tags aligned and also indented properly. For example: 
```
<h3> .....
<---><div>
<--->       Jeremy Blum '13 
<---></div>
</h3>
```

Or, keep it on one line, but still indent the inner div: 
```
<h3> .....
<---><div> Jeremy Blum '13 </div>
</h3>
```